src/components/settings/SettingsAppearanceTab/index.jsx

```diff
@@ -53,11 +126,20 @@ export default function SettingsAppearanceTab({
           'image/*': ['.png'],
         }}
       />
+      )}
+      {uploadedFile && (
+      <p className="image-preview">
+        <Image
+          src={window.URL.createObjectURL(uploadedFile)}
```

Comment:
nit: `global` versus `window` _just in case_ this code ever runs in an environment where `window` doesn't exist.